daemons/fenced/fenced_commands.c

```diff
@@ -3190,13 +3187,12 @@ remove_relay_op(xmlNode * request)
 
     /* If the operation to be deleted is registered as a duplicate, delete the registration. */
     while (g_hash_table_iter_next(&iter, NULL, (void **)&list_op)) {
-        GList *dup_iter = NULL;
-
         if (list_op == relay_op) {
             continue;
         }
 
-        for (dup_iter = list_op->duplicates; dup_iter != NULL; dup_iter = dup_iter->next) {
+        for (const GList *dup_iter = list_op->duplicates; dup_iter != NULL;
```

Comment:
This shouldn't be `const` because we may modify the `duplicates` list via `dup_iter`. (It is ALLOWED to be const, but that's the standard we've been following for clarity.)